autotst/conformer/systematic.py

```diff
@@ -213,25 +217,42 @@ def opt_conf(conformer, calculator, i):
 
             calculator.atoms = conformer.ase_molecule
 
-        from ase.constraints import FixBondLengths
-        c = FixBondLengths(labels)
-        conformer.ase_molecule.set_constraint(c)
-
         conformer.ase_molecule.set_calculator(calculator)
-
         opt = BFGS(conformer.ase_molecule, logfile=None)
-        opt.run(fmax=0.1)
+
+        if type == 'species':
+            try:
+                opt.run(steps=1000)
+            except RuntimeError:
+                logging.info("Optimization failed...we will use the unconverged geometry")
+                pass
+        
+        if type == 'ts':
+            c = FixBondLengths(labels)
+            conformer.ase_molecule.set_constraint(c)
+            try:
+                opt.run(fmax=0.20, steps=1e6)
```

Comment:
Yes, optimizing based on fmax for TS is not a great method.  I did try to optimize to a percentage of the initial fmax in the reaction center for each conformer, but then the different TS conformers have different fmax convergence criteria, which is not good.  I found that an arbitrary fmax of 0.2 works pretty well.

Reply:
In most cases, the molecule should require less steps.  I capped the TS opt to 1e6 to save computational time

Reply:
Okay, I'm fine leaving the fmax as 0.2 for now... I'm just not terribly jazzed about the fact that this number is arbitrary. I think this can be a decenter temporary fix until we transition to py3 and start implementing Sella. Sella doesn't have any fmax or nsteps feature so it always runs to the minima.